tests/test_utils_io.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+import genet.utils.io
+import pytest
+from geopandas import GeoDataFrame
+from shapely import Point
+
+
+def test_throws_error_when_filetype_is_not_supported():
+    with pytest.raises(RuntimeError) as e:
+        genet.utils.io.check_file_type_is_supported("hello")
+
+    assert "is not a supported file type" in str(e.value)
+
+
+def test_saving_geodataframe_with_missing_geometry_produces_file(tmpdir):
```

Comment:
Maybe add another test that checks the expected behaviour if the gdf is completely empty (i.e. no file exists)?